### ISSUE TITLE
Verifying an email address after mutating a user

### ISSUE THREAD

**Issue by rjgux** [jamesedmonston/graphql-authentication#90]
On the front-end of our site we're allowing users to update their email address via the following mutation -

```
mutation ($email: String!) {
  updateViewer(email: $email) {
    id
    email
  }
}
```

However, this doesn't trigger Craft's email verification system. The new email address and user is saved automatically without requiring any verification steps and doesn't trigger the `verify_new_email` system message in Craft.

Is this possible through the `updateViewer` mutation or by using this plugin?

Thanks!

**Comment by rjgux** [comment]
I'm not sure if this is a bug with the current implementation or whether it's a feature that needs adding, but we would happy to try and help out and contribute.

**Comment by jamesedmonston** [comment]
@rjgux It's mainly an oversight in my original implementation of the `updateViewer` plugin, where it updates _everything_ without any validation. I'd be happy to grant you a license for a PR!

I'm going to be fixing bugs and updating for Craft 4 very shortly either way, so no stress!

**Comment by jamesedmonston** [comment]
@rjgux released in 2.0!

**Comment by howells** [comment]
@jamesedmonston sorry to bump a closed issue but I've just upgraded from Craft 3 and so I'm now seeing verification emails. However, I don't particularly want these, especially since they are triggered when the email address doesn't actually change. I can't find where I can disable this in the Craft docs at all - is there a way I can it within GQL Auth?
